scripts/web_ui.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+if [ ! -f deployment/env ]; then
+    echo "Error: Deployment environment file not found. Run 'make setup-local' or 'make setup-azure' first."
+    exit 1
+fi
+source ./deployment/env
+
+if ! kubectl get namespace "${BUTTERCUP_NAMESPACE:-crs}" >/dev/null 2>&1; then
+    echo "Error: CRS namespace not found. Deploy first with 'make deploy'."
+    exit 1
+fi
+
+echo "Opening web UI..."
+
+if [ "${TAILSCALE_ENABLED:-false}" = "true" ]; then
+    # Use Tailscale domain
+    if [ -z "${TAILSCALE_DOMAIN}" ]; then
+        echo "Error: TAILSCALE_ENABLED is true but TAILSCALE_DOMAIN is not set"
+        exit 1
+    fi
+    CRS_HOSTNAME="${CRS_HOSTNAME:-buttercup-ui}"
+    UI_URL="http://${CRS_HOSTNAME}.${TAILSCALE_DOMAIN}"
+else
+    # Use local port-forward
+    kubectl port-forward -n "${BUTTERCUP_NAMESPACE:-crs}" service/buttercup-ui 31323:1323 &
```

Comment:
Just occurred to me that we might want to record the pid for this case and have the `web-ui.sh` wait for it or, otherwise kill it on trap (e.g. Ctrl+C) to ensure we don't leave dangling port-forwards. What do you think? Overkill?

Reply:
I'm not sure that's possible, as the script terminates and the port-forwarding should still be up and running to reach the webui.

Reply:
That's my point, keep the script running waiting for the port-forward to end, or for the user to exit it and then kill the port-forward.

Reply:
I see. We can do that yeah! i'll fix it asap